src/index/index.tsx

```diff
@@ -82,45 +82,59 @@ const Home = ({
       </section>
 
       <section className={style.join}>
-        <h2>Bli en variant!</h2>
-        <p>
-          For å kunne lage gode løsninger trenger vi forskjellige mennesker med
-          forskjellig perspektiv. Vi trenger flinke folk med engasjerte stemmer.
-        </p>
+        <h2 className={style.join_title}>Bli en Variant!</h2>
 
         <p>
-          <strong>Det betyr deg.</strong>
+          For å kunne lage gode løsninger trenger vi forskjellige mennesker med
+          forskjellig perspektiv. Vi trenger flinke folk med engasjerte stemmer!
         </p>
-        <ButtonNextLink href="/jobs">Bli en variant</ButtonNextLink>
 
-        <p>
-          Vi har mange{' '}
-          <Link href="/ansatte">
-            <a title="Ansattoversikt">flotte kolleger</a>
-          </Link>
-          , et{' '}
-          <a
-            href="https://www.google.com/maps/place/Varianthuset/@63.4328051,10.397323,17z/data=!3m1!4b1!4m5!3m4!1s0x466d312df4ea1347:0xf63e949e041942ee!8m2!3d63.4328051!4d10.3995117"
-            rel="noopener"
-            target="_blank"
-            title="Kart til Varianthuset"
-          >
-            fint hus
-          </a>
-          ,{' '}
-          <a href="https://handbook.variant.no" title="Variant Håndbok">
-            gode (og åpne) vilkår
-          </a>{' '}
-          og{' '}
-          <Link href="/kalkulator">
-            <a title="Lønnskalkulator">god lønn</a>
-          </Link>
-          . Ikke minst har vi mange{' '}
-          <a href="#prosjekter" title="Prosjekteksempler">
-            spennende prosjekter
-          </a>
-          .
-        </p>
+        <p>Kanskje det er deg?</p>
+
+        <div className={style.join_info}>
+          <div>
+            <p>
+              Vi holder til i superfine lokaler i Trondheim, Oslo og Bergen. I
+              tillegg har vi noen små, sjarmerende satelittkontor i Molde og
+              Harstad.
+            </p>
+            <p>
+              Der har vi en haug av{' '}
+              <Link href="/ansatte">
+                <a title="Ansattoversikt">flotte kolleger</a>
+              </Link>
+              ,{' '}
+              <a href="https://handbook.variant.no" title="Variant Håndbok">
+                gode (og selvfølgelig åpne) vilkår
+              </a>{' '}
+              og{' '}
+              <Link href="/kalkulator">
```

Comment:
Her linkes til kalkulator, men skulle vi heller ha linka til https://handbook.variant.no/#l%C3%B8nn som nevner bonus, pensjon og andre goder? @oddsve @mikaelbr? Ikke blocker for PR altså, mest et åpent spørsmål jeg tenkte på nå

Reply:
Fikk høre fra Truls at dette kommer i neste versjon av kalkulator, så da anser jeg det som resolved!

Reply:
Når #225 blir ferdig så kommer kalkulatoren til inneholde info om bonus, pensjon, og all the other good stuff :D